python/lsst/ts/mtmount/config_schema.py

```diff
@@ -45,7 +45,15 @@
     type: number
     exclusiveMinimum: 0
   ack_timeout:
-    description: Time limit for reading a command acknowledgement from the TCP/IP interface (sec)
+    description: >-
+        Time limit for reading a command acknowledgement from the TCP/IP interface
+        for most commands (sec)
+    type: number
+    exclusiveMinimum: 0
+  ack_timeout_track_target:
+    description: >-
+        Time limit for reading a command acknowledgement from the TCP/IP interface
+        for main axis and camera cable wrap track target commands (sec)
```

Comment:
You wrote in the Jira ticket that the TMA is supposed to ack all commands quickly. Does it? If yes then I'd suggest to use a single ack_timeout. If it doesn't then I'd suggest to talk to Tekniker and ask them to make sure that the TMA follows the requirement of ack-ing quickly and, once that's done, refactor to a single ack_timeout.

Reply:
And nitpicking here but all description sentences are supposed to end in a `.` and barely any in this schema do.